packit/upstream.py

```diff
@@ -766,3 +768,78 @@ def koji_build(
             print_live=True,
         )
         return out
+
+    def create_rpm(self, rpm_path, rpm_dir: Union[str, Path] = None):
```

Comment:
can we share the code here with create_srpm?

Reply:
We use now different exceptions and messages, but otherwise I can change it